test/regression/services/stats_spec.js

```diff
@@ -0,0 +1,603 @@
+const supertest = require('supertest');
+const _ = require('lodash');
+const moment = require('moment');
+const should = require('should');
+const Bluebird = require('bluebird');
+
+const testUtils = require('../../utils');
+const localUtils = require('../api/canary/admin/utils');
+const config = require('../../../core/shared/config');
+const models = require('../../../core/server/models');
+const db = require('../../../core/server/data/db');
+const ghost = testUtils.startGhost;
+const authorContext = testUtils.context.owner;
+let request;
+
+function sleep(ms) {
+    return Bluebird.delay(ms);
+}
+
+function post() {
+    return _.clone(testUtils.DataGenerator.forModel.posts[0]);
+}
+
+const PREVENT_FLAKINESS_TIME = 400;
```

Comment:
I tried to reduce the number. But it causes flakiness. 

Maybe we can add an event at the end of the `updateGlobalTemplateOptions` and use it in the test.